### DISCUSSION TITLE
SO Ubuntu Server 20.04 Install

### DISCUSSION THREAD

**Discussion by dummyaccnt80** [Security-Onion-Solutions/securityonion#2669] [Unsupported Versions] [answered]
Is the new SO only compatible with Ubuntu Server 18.04? Planning to use Ubuntu Server 20.04, anyone tried it?

**Comment by trb143** [comment]
The installer seems to be hard-wired to the 18.04 salt repository so you may have issues!

**Reply by ServSlack** [reply]
Hello Guys,

I install SecurityOnion in Ubuntu 20.04 doing some little changes on Setup script, but comparing the services installed between 18.04 and 20.04 I saw that are missing this two containers.

Can you please help me to reinstall those containers ?

so-cortex
so-zeek

**Comment by TOoSmOotH** [comment, marked as answer]
It is not supported due to a missing package in 20.04 which is why it is locked to 18.04. #601 is the issue that is tracking it.
